desktop/core/src/desktop/js/jquery/plugins/jquery.filechooser.js

```diff
@@ -92,6 +92,23 @@ const defaults = {
         name: 'S3'
       }
     },
+    gs: {
+      scheme: 'gs',
+      root: 'gs://',
+      home: '/?default_gs_home',
+      icon: {
+        svg: {
+          brand: '#hi-adls',
```

Comment:
adls?

Reply:
Just a placeholder and branding will be done in separate PR.